src/components/grid-aware/ArticleSpotlightCard/ArticleSpotlightCard.module.css

```diff
@@ -76,6 +76,11 @@
   margin-top: 20px;
 }
 
+.subtitle {
+  color: var(--color-gray-600);
+  font: var(--font-body-medium);
```

Comment:
In the design, it looks like the font is actually `--font-subtitle`, not `--font-body-medium`.